dist/pptxgen.d.ts

```diff
@@ -34,67 +34,325 @@ declare class PptxGenJS {
 
 declare namespace PptxGenJS {
   const version: string;
-  export type ChartType = "AREA" | "BAR" | "BUBBLE" | "DOUGHNUT" | "LINE" | "PIE" | "RADAR" | "SCATTER";
-  export type JsZipOutputType = "arraybuffer" | "base64" | "binarystring" | "blob" | "nodebuffer" | "uint8array";
-  export type LayoutName = "LAYOUT_4x3" | "LAYOUT_16x9" | "LAYOUT_16x10" | "LAYOUT_WIDE";
+  export type ChartName = 'AREA' | 'BAR' | 'BUBBLE' | 'DOUGHNUT' | 'LINE' | 'PIE' | 'RADAR' | 'SCATTER'
+  export type ChartType = ChartName | { displayName: string; name: string };
+  export type JsZipOutputType = 'arraybuffer' | 'base64' | 'binarystring' | 'blob' | 'nodebuffer' | 'uint8array';
+  export type LayoutName = 'LAYOUT_4x3' | 'LAYOUT_16x9' | 'LAYOUT_16x10' | 'LAYOUT_WIDE';
   export interface Layout {
-    name: string;
+    name?: string;
     width: number;
     height: number;
   }
-  export type Color = string;
+  export type HexColor = string; // should match /^[0-9a-fA-F]{6}$/
+  export type ThemeColor = 'tx1' | 'tx2' | 'bg1' | 'bg2' | 'accent1' | 'accent2' | 'accent3' | 'accent4' | 'accent5' | 'accent6';
+  export type Color = HexColor | ThemeColor;
   export type Coord = number | string; // string is in form 'n%'
 
+  export type HAlign = 'left' | 'center' | 'right';
+  export type VAlign = 'top' | 'middle' | 'bottom';
+
+  export type ChartTimeUnit = 'days' | 'months' | 'years';
+  export type ChartAxisOrientation = 'minMax' | 'maxMin';
+
+  export type ChartLabelPos = 'low' | 'high' | 'nextTo';
+
   export interface CommonOptions {
     x?: Coord;
     y?: Coord;
     w?: Coord;
     h?: Coord;
   }
-  export interface DataOrPath {
-    // Exactly one must be set
-    data?: string;
-    path?: string;
-  }
-  export interface ImageOptions extends CommonOptions, DataOrPath {
+
+  export type DataOrPath = { data: string; path?: undefined } | { data?: undefined; path: string };
+
+  export interface ImageBaseOptions extends CommonOptions {
```

Comment:
Hello @iota-pi 
Thank you for improving TS definition.

Could you insert Hyperlink and ImageSizingOptions in your Pull Request please ?
`
  export interface ImageBaseOptions extends CommonOptions {
    hyperlink?: Hyperlink;
    rounding?: boolean;
    sizing?: ImageSizingOptions;
  }

  export interface ImageSizingOptions extends CommonOptions {
    type?: 'cover' | 'contain' | 'crop';
  }

  export type Hyperlink = ({ url: string; slide?: undefined } | { slide: number; url?: undefined }) & { tooltip?: string };
`